### ISSUE TITLE
Colors are not synchronised

### ISSUE THREAD

**Issue by nbraud** [c3subtitles/L2S2#89]
The colors assigned to an angel change depending on the client, which makes synchronisation more confusing.

Also, some colors are too similar.

**Comment by 4n08o** [comment]
I would prefer to have the colors local, i. e. I choose my color scheme and an other subtitler choose his/her colors.  Meaning a local assignment of colors according to personal preferences for my own color and the colors for the other subtitlers.


Subtitler 1: red, yellow, green  (own, subtitler 2, subtitler 3,...)
Subtitler 2: blue, green, yellow (own, subtitler 1, subtitler 3,...)
...

**Comment by bjelline** [comment]
quick fix for colors being to simliar:

in src/Reducers/room.js

    const colors: List = List(['#aec7e8', '#ffbb78', '#98df8a', '#ff9896', '#c5b0d5', '#c49c94', '#f7b6d2', '#c7c7c7', '#dbdb8d', '#9edae5']);

**Comment by patrickhanft** [comment]
@4n08o I disagree. We had it quite often, that someone said: "I will type after the purple one" and only quite some time we recognized, that everybody was referring to somebody else.

**Comment by marudor** [comment]
Yes, synchronized colors should be used.
Sadly this is something non trivial.
It is definitively on my todo for the next time.
